File: 3er CURSO/2-CUATRI/SD/PRACTICAS/Práctica 3 (C#)/Cliente_SW_Sudoku/Cliente_SW_Sudoku/ListaSudoku.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

using Cliente_SW_Sudoku.GestorJuegos_Web;

namespace Cliente_SW_Sudoku
{
    class ListaSudoku
    {
        private int Codigo;
        private TipoDificultad Dificultad;

        public ListaSudoku() { }

        public TipoDificultad GetDificultad()
        {
            return Dificultad;
        }

        public int getCodigo()
        {
            return Codigo;
        }

        public ListaSudoku(int c, TipoDificultad dif)
        {
            Codigo = c;
            Dificultad = dif;
        }
    }
}

File: 3er CURSO/2-CUATRI/SD/PRACTICAS/Práctica 2_MOD/Cliente_Sudoku/src/cliente_sudoku/ListaSudoku.java

package cliente_sudoku;

import libreria_sudoku.TipoDificultad;

/**
 *
 * @author Ismael Da Palma Fernández
 */
public class ListaSudoku {
    int Codigo;
    TipoDificultad Dificultad;

    public ListaSudoku() {
    }
    
    public ListaSudoku(int c, TipoDificultad dif) {
        Codigo = c;
        Dificultad = dif;
    }
    
    
}
